docs/build.js

```diff
@@ -11,18 +11,24 @@ async function build() {
 
 	const start = Date.now();
 
+	const assetsPath = path.resolve(__dirname, '../app/public/img/docs');
 	const distPath = path.resolve(__dirname, './dist');
 
 	await rimraf(distPath);
 
 	const tree = dirTree('.', { extensions: /\.md/, exclude: /(dist|node_modules)/ });
 
+	await fse.ensureDir(assetsPath);
 	await fse.ensureDir(distPath);
 
 	await fse.writeJSON('./dist/index.json', tree);
 
 	await copyfiles(['./**/*.md', distPath], { exclude: './node_modules/**/*.*' });
 	await copyfiles(['./assets/**/*.*', distPath], { exclude: './node_modules/**/*.*' });
+	
+	console.log('Copying docs assets to:', assetsPath);
+	
+	await copyfiles(['./assets/**/*.*', assetsPath], { exclude: './node_modules/**/*.*' });
```

Comment:
Can we move this build logic into the build configuration of the app instead of the docs? I want to make sure we keep the different responsibilities separated per package (eg the app relies on the docs folder for source, not the other way around)

Reply:
Will do...
Do you mind confirming the app build script, or do I need to create one?

Reply:
We most likely have to add a "pre-build" script that copies the files over, there's no build file or anything yet (all abstracted away in vue-cli)

Reply:
I welcome any guidance or recommendations on this.

Reply:
Yeah I'll take a swing at it 👍🏻